platform/FEATURE_EXPERIMENTAL_API/FEATURE_PSA/TARGET_MBED_PSA_SRV/TESTS/attestation/test/CMakeLists.txt

```diff
@@ -10,9 +10,17 @@ include(${MBED_PATH}/tools/cmake/mbed_greentea.cmake)
 
 project(${TEST_TARGET})
 
+if(MBED_TEST_BAREMETAL)
+    set(TEST_SKIPPED "RTOS required")
+elseif(NOT ("PSA" IN_LIST MBED_TARGET_LABELS AND "EXPERIMENTAL_API" IN_LIST MBED_TARGET_LABELS))
```

Comment:
For now it's here, but when we port this test to CTest like in #14892, this check may not be needed anymore if the directory is not added